ms2pip/ms2pipC.py

```diff
@@ -883,8 +873,11 @@ def __init__(
             if config_file is None:
                 raise MissingConfigurationError()
             else:
-                self.params = load_configfile(config_file)
+                self.config_parser = ConfigParser(filepath=config_file)
+                self.config_parser.load()
+                self.params = self.config_parser.config['ms2pip']
         else:
+            self.config_parser = None
```

Comment:
Another option could be to always pass the config as a params dict to the runner, and move the config parser logic to main.

Reply:
Indeed. Removed `config_file` argument from MS2PIP class, and moved all config parser logic to `__main__.py`. One consequence of this (required for deeplc integration, and fasta2speclib in the future) is that params is now a dict with `{"ms2pip": <ms2pip_params>}` or `{"ms2pip": <ms2pip_params>, "deeplc": <deeplc_params>}`